content/docs/dvclive/dvclive-with-dvc.md

```diff
@@ -0,0 +1,147 @@
+# Dvclive with DVC
+
+Even though dvclive does not require DVC to function properly, it includes a lot
+of integrations with DVC that you might find valuable. In this section we will
+modify the [basic usage example](/doc/dvclive/usage) to see how DVC can
+cooperate with dvclive.
+
+Let's use the code prepared in previous example and try to make it work with
+dvc. Training file `train.py` content:
+
+```python
+from keras.datasets import mnist
+from keras.models import Sequential
+from keras.layers.core import Dense, Activation
+from keras.utils import np_utils
+
+def load_data():
+    (x_train, y_train), (x_test, y_test) = mnist.load_data()
+
+    x_train = x_train.reshape(60000, 784)
+    x_test = x_test.reshape(10000, 784)
+    x_train = x_train.astype('float32')
+    x_test = x_test.astype('float32')
+    x_train /= 255
+    x_test /= 255
+    classes = 10
+    y_train = np_utils.to_categorical(y_train, classes)
+    y_test = np_utils.to_categorical(y_test, classes)
+    return (x_train, y_train), (x_test, y_test)
+
+def get_model():
+    model = Sequential()
+    model.add(Dense(512, input_dim=784))
+    model.add(Activation('relu'))
+
+    model.add(Dense(10, input_dim=512))
+
+    model.add(Activation('softmax'))
+
+    model.compile(loss='categorical_crossentropy',
+    metrics=['accuracy'], optimizer='sgd')
+    return model
+
+
+from keras.callbacks import Callback
+import dvclive
```

Comment:
@pared can these be on the top of the file for readability?

Reply:
Also, shouldn't classes (class MetricsCallback) be defined before functions?

Reply:
It should, I thought that including it just above "execution" part makes it easier to copy-paste for user

Reply:
Hmm I missed this. I'll send a PR with what I meant and request your review @pared 🙂